### DISCUSSION TITLE
Functionality to rcreate already filtered html file

### DISCUSSION THREAD

**Discussion by taumeister** [EvotecIT/PSWriteHTML#245] [Ideas]
first of all, thanks for your wonderful work on these modules, we use them a lot and they are a great peace of help.
Basically there is almost no wish, except one thing that  my team and I have.

Would it be possible to create a functionality/and button to create a new html file, from the actual filtered one, that again has all the functions and features included?

What we really often have is the case, that we for example have a big big list of computerobjects, filter them properly and then want to save them to a new html file, BUT then filtered and then with alle the features to filter and convert, like the origin html.

We could send this new file and the receiver would see the for him filtered list without loosing functionality and data.

Greetings and keep on the good work

**Comment by PrzemyslawKlys** [comment]
I don't think it's possible to do. What you're asking is to add a button next to the table that currently has CSV/Excel and when you press it, it exports to the very same HTML but with only data that was selected. It's not possible. It's only possible to export "selected" data to Excel, CSV and PDF. 

What is possible is to pre-create in one go multiple HTML pages with different data. But it's all done on PowerShell level, not on HTML level. So any filtering needs to be applied on PS.
